### PR TITLE
CASMPET-5640 Corrected VCS Password Update instructions (release/1.2)

### PR DESCRIPTION
**Pull request by atifsyedali**
## Summary and Scope

_Summarize what has changed. Explain why this PR is necessary. What is impacted? Is this a new feature, critical bug fix, etc?_

Corrected VCS password update instructions. No code changes.

_Is this change backwards incompatible, backwards compatible, or a backwards compatible bugfix?_

VCS password update instructions have stayed the same since 1.0 to mainline dev so not expecting incompatibilities.

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [CASMPET-5640](https://jira-pro.its.hpecorp.net:8443/browse/CASMPET-5640)
* Change will also be needed in `release/1.0` and `main`
* ~Future work required by [issue id](issue link)~
* ~Documentation changes required in [issue id](issue link)~
* ~Merge with/before/after `<insert PR URL here>`~

## Testing

_List the environments in which these changes were tested._

### Tested on:

  * `fanta`

### Test description:

_How were the changes tested and success verified? If schema changes were part of this change, how were those handled in your upgrade/downgrade testing?_

1. Followed instructions to change VCS password and then verified the change by logging into VCS.
2. Change the VCS password back.

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)? No
- Were continuous integration tests run? If not, why?  No
- Was upgrade tested? If not, why?  No
- Was downgrade tested? If not, why?  No
- Were new tests (or test issues/Jiras) created for this change?  No

## Risks and Mitigations

_Are there known issues with these changes? Any other special considerations?_

No other known issues.

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: operations/configuration_management/Version_Control_Service_VCS.md [hunk 1]
```diff
@@ -4,43 +4,208 @@ The Version Control Service \(VCS\) includes a web interface for repository mana
...
+1. Save an updated copy of `customizations.yaml` to the `site-init` secret in the `loftsman` Kubernetes namespace.
+
+    ```bash
+    ncn# CUSTOMIZATIONS=$(base64 < customizations.yaml  | tr -d '\n')
+    ncn# kubectl get secrets -n loftsman site-init -o json |
+            jq ".data.\"customizations.yaml\" |= \"$CUSTOMIZATIONS\"" |
+            kubectl apply -f -
+    ```
 
 ## Access the `cray` Gitea Organization
```
Comment (mharding-hpe) [inline operations/configuration_management/Version_Control_Service_VCS.md:203]:
```suggestion
## Access the `cray` Gitea organization
```

File: operations/configuration_management/Version_Control_Service_VCS.md [hunk 2]
```diff
@@ -70,7 +235,9 @@ https://vcs.SHASTA_CLUSTER_DNS_NAME/vcs/org/cray/teams
 
 ## Backup and Restore Data
```
Comment (mharding-hpe) [inline operations/configuration_management/Version_Control_Service_VCS.md:236]:
```suggestion
## Backup and restore data
```

File: operations/configuration_management/Version_Control_Service_VCS.md [hunk 3]
```diff
@@ -70,7 +235,9 @@ https://vcs.SHASTA_CLUSTER_DNS_NAME/vcs/org/cray/teams
...
 
-Data for Gitea is stored in two places. Git content is stored directly in a PVC, while structural data, such as Gitea users and the list and attributes of repos, is stored in a Postgres database. Because of this, both sources must be backed up and restored together.
+Data for Gitea is stored in two places. Git content is stored directly in a PVC, while structural data, such as Gitea users
+and the list and attributes of repos, is stored in a Postgres database. Because of this, both sources must be backed up
+and restored together.
 
 ### Backup Postgres Data
```
Comment (mharding-hpe) [inline operations/configuration_management/Version_Control_Service_VCS.md:242]:
```suggestion
### Backup Postgres data
```

File: operations/configuration_management/Version_Control_Service_VCS.md [hunk 4]
```diff
@@ -123,11 +290,11 @@ Data for Gitea is stored in two places. Git content is stored directly in a PVC,
     done
     ```
 
-4. Edit the manifest file to remove creationTimestamp, resourceVersion, selfLink, uid for each entry. Then, copy all files to a safe location.
+4. Edit the manifest file to remove `creationTimestamp`, `resourceVersion`, `selfLink`, `uid` for each entry. Then, copy all files to a safe location.
 
 ### Backup PVC Data
```
Comment (mharding-hpe) [inline operations/configuration_management/Version_Control_Service_VCS.md:295]:
```suggestion
### Backup PVC data
```

File: operations/configuration_management/Version_Control_Service_VCS.md [hunk 5]
```diff
@@ -139,11 +306,11 @@ ncn# kubectl -n services cp ${POD}:vcs.tar ./vcs.tar
 
 ### Restore Postgres Data
```
Comment (mharding-hpe) [inline operations/configuration_management/Version_Control_Service_VCS.md:307]:
```suggestion
### Restore Postgres data
```

File: operations/configuration_management/Version_Control_Service_VCS.md [hunk 6]
```diff
@@ -139,11 +306,11 @@ ncn# kubectl -n services cp ${POD}:vcs.tar ./vcs.tar
...
 
-Restoring VCS from Postgres is documented here: [Restore_Postgres.md](../../operations/kubernetes/Restore_Postgres.md#restore-postgres-for-vcs)
+Restoring VCS from Postgres is documented here: [Restore Postgres](../../operations/kubernetes/Restore_Postgres.md#restore-postgres-for-vcs)
 
 ### Restore PVC Data
```
Comment (mharding-hpe) [inline operations/configuration_management/Version_Control_Service_VCS.md:311]:
```suggestion
### Restore PVC data
```

File: operations/configuration_management/Version_Control_Service_VCS.md [hunk 7]
```diff
@@ -156,7 +323,10 @@ ncn# kubectl -n services rollout restart deployment gitea-vcs
 
 ### Alternative Backup/Restore Strategy
```
Comment (mharding-hpe) [inline operations/configuration_management/Version_Control_Service_VCS.md:324]:
```suggestion
### Alternative backup/restore strategy
```

**Review by mharding-hpe — CHANGES_REQUESTED** [review]

**Comment by atifsyedali** [comment]
Thanks for the review @mharding-hpe!

**Review by mharding-hpe — APPROVED** [review]

**Review by nrockershousen — APPROVED** [review]

**Comment by atifsyedali** [comment]
I don't seem to have permissions to merge. Can someone please squash and merge this?

File: operations/configuration_management/Version_Control_Service_VCS.md [hunk 8]
```diff
@@ -4,43 +4,208 @@ The Version Control Service \(VCS\) includes a web interface for repository mana
...
+    ```
+
+1. Edit the `sysmgmt.yaml` to only include the `gitea` chart and all its current data.
+
+   Example:
+
+    ```yaml
+    apiVersion: manifests/v1beta1
+      metadata:
+        name: sysmgmt
```
[the remark this answers is not in the document]
Comment (atifsyedali) [reply operations/configuration_management/Version_Control_Service_VCS.md:166]: @kburns-hpe Made changes as you suggested. Can you please review? Thanks

[the remark this answers is not in the document]
Comment (atifsyedali) [reply operations/configuration_management/Version_Control_Service_VCS.md:171]: Hi @kburns-hpe Any chance you can have a look/approve at the changes today? Thanks.

**Review by kburns-hpe — APPROVED** [review]

**Pull request merged**